spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/ExpireSnapshotsSparkAction.java

```diff
@@ -164,8 +166,9 @@ public Dataset<Row> expire() {
    */
   public Dataset<FileInfo> expireFiles() {
     if (expiredFileDS == null) {
-      // fetch metadata before expiration
-      Dataset<FileInfo> originalFileDS = validFileDS(ops.current());
+
+      // Save old metadata
+      TableMetadata originalTable = ops.current();
```

Comment:
nit: What about calling it `originalMetadata` since it is `TableMetadata`, not `Table`?

Reply:
Done